pkg/reconciler/testing/v1/eventpolicy.go

```diff
@@ -44,25 +44,72 @@ func NewEventPolicy(name, namespace string, o ...EventPolicyOption) *v1alpha1.Ev
 	return ep
 }
 
-func WithInitEventPolicyConditions(et *v1alpha1.EventPolicy) {
-	et.Status.InitializeConditions()
+func WithInitEventPolicyConditions(ep *v1alpha1.EventPolicy) {
+	ep.Status.InitializeConditions()
 }
 
-func WithReadyEventPolicyCondition(ep *v1alpha1.EventPolicy) {
-	ep.Status.Conditions = []apis.Condition{
-		{
-			Type:   v1alpha1.EventPolicyConditionReady,
+func WithTrueAuthenticationEnabledCondition(ep *v1alpha1.EventPolicy) {
```

Comment:
```suggestion
func WithEventPolicyAuthenticationEnabledCondition(ep *v1alpha1.EventPolicy) {
```
I think we should add the CR name, as we have many functions in this package

Reply:
Thanks, I updated it in the newer commit.